main.tf

```
@@ -19,6 +19,8 @@ locals {
     var.associate_public_ip_address && var.assign_eip_address && module.this.enabled ?
     local.eip_public_dns : join("", aws_instance.default.*.public_dns)
   )
+  ssm_path_log_bucket_enabled = module.this.enabled && var.ssm_patch_manager_enabled && var.ssm_patch_manager_s3_log_bucket != ""
+  ssm_policy                  = var.ssm_patch_manager_iam_policy == "" ? "arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore" : var.ssm_patch_manager_iam_policy
```

```suggestion
  ssm_policy                  = var.ssm_patch_manager_iam_policy == null || var.ssm_patch_manager_iam_policy == "" ? "arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore" : var.ssm_patch_manager_iam_policy
```

the var is set to `null`